app/audit/Utilities.scala

```diff
@@ -35,6 +35,15 @@ object Utilities {
     ("credId", user.credId) ++
     userType(user.userType)
 
+  def userAuditDetailsNino(user: MtdItUserBase[_]): JsObject = Json.obj(
+    "nino" -> user.nino,
+    "mtditid" -> user.mtditid
+  ) ++
+    ("agentReferenceNumber", user.arn) ++
```

Comment:
dupe code here from above - can you extract a common private method eg getBaseDetails (without nino fields) and then apply either nino or nationalinsurancenumber depending on the called method (ie userAuditDetails or userAuditdetailsNino)?

Reply:
resolved in recent commit